### PR TITLE
Update README.md fo 2.5

### PR DESCRIPTION
**Pull request by ManfeiBai**
Update README.md fo 2.5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,10 +204,13 @@ The torch wheel version `2.6.0.dev20240925+cpu` can be found at https://download
...
 | 2.5 (CUDA 12.1 + Python 3.10) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0-cp310-cp310-manylinux_2_28_x86_64.whl` |
 | 2.5 (CUDA 12.1 + Python 3.11) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0-cp311-cp311-manylinux_2_28_x86_64.whl` |
-| 2.4 (CUDA 12.1 + Python 3.8) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.4.0-cp38-cp38-manylinux_2_28_x86_64.whl` |
+| 2.5 (CUDA 12.4 + Python 3.9) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.4/torch_xla-2.5.0-cp39-cp39-manylinux_2_28_x86_64.whl` |
+| 2.5 (CUDA 12.4 + Python 3.10) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.4/torch_xla-2.5.0-cp310-cp310-manylinux_2_28_x86_64.whl` |
+| 2.5 (CUDA 12.4 + Python 3.11) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.4/torch_xla-2.5.0-cp311-cp311-manylinux_2_28_x86_64.whl` |
+| 2.4 (CUDA 12.1 + Python 3.9) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.4.0-cp39-cp39-manylinux_2_28_x86_64.whl` |
```
Comment (JackCaoG) [inline README.md:213]: did you verify these wheel exist?

Comment (ManfeiBai) [reply README.md:213]: yes, verified

**Comment by ManfeiBai** [comment]
Due to only change readme.md, CI failed is not related

File: README.md [hunk 2]
```diff
@@ -204,10 +204,13 @@ The torch wheel version `2.6.0.dev20240925+cpu` can be found at https://download
 
 | Version | GPU Wheel |
 | --- | ----------- |
-| 2.5 (CUDA 12.1 + Python 3.8) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0-cp38-cp38-manylinux_2_28_x86_64.whl` |
+| 2.5 (CUDA 12.1 + Python 3.9) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0-cp39-cp39-manylinux_2_28_x86_64.whl` |
 | 2.5 (CUDA 12.1 + Python 3.10) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0-cp310-cp310-manylinux_2_28_x86_64.whl` |
 | 2.5 (CUDA 12.1 + Python 3.11) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0-cp311-cp311-manylinux_2_28_x86_64.whl` |
-| 2.4 (CUDA 12.1 + Python 3.8) | `https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.4.0-cp38-cp38-manylinux_2_28_x86_64.whl` |
```
Comment (tengyifei) [inline README.md:210]: Did the 2.4 URL never work in the first place so now you remove it? I visited https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.4.0-cp38-cp38-manylinux_2_28_x86_64.whl and got 404.

**Review by tengyifei — APPROVED** [review]
I also verified these links work.

File: README.md (hunk 2 above)
Comment (ManfeiBai) [reply README.md:210]: when I check, I found no trigger for py38 for 2.4.0, so update it to exitsing trigger py39 for 2.4.0, and we don't have this wheel too, its more like a cleanup

**Pull request merged**